assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_metrics/run.py

```diff
@@ -86,11 +86,7 @@ def run():
     parser.add_argument("--coherence_passrate_threshold", type=float, default=0.7)
 
     args = parser.parse_args()
-    histogram_df = try_read_mltable_in_spark_with_warning(args.annotation_histogram, "annotation_histogram")
-
-    if not histogram_df:
-        print("No histogram to annotate. Skipping computing annotation metrics.")
-        return
+    histogram_df = try_read_mltable_in_spark_with_error(args.annotation_histogram, "annotation_histogram")
```

Comment:
Can you update the read_mltable_in_spark call in annotation_compute_histogram/run.py too ?

Reply:
I searched try_read_mltable_in_spark, so missed this one, sure, will update it too.